poc/vdaf_mastic.py

```diff
@@ -158,6 +163,8 @@ def prep_init(cls, verify_key, agg_id, agg_param,
                                                         cls.ROOT_PROOF,
                                                         nonce)
             meas_share = vec_add(out_share[0], out_share[1])
+            # As a consequence of VIDPF path verifiability,
+            # meas_share contains the value of the root node in the VIDPF tree
```

Comment:
* Generally block comments precede the code they're commenting on.
* I think we could merge this into the block comment above `eval()`.